maintenance/wikia/WikiaMaps/removeImapTags.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+/**
+ * Maintenance script to remove <imap /> tags from articles
+ *
+ * @usage SERVER_ID=203236 php removeImapTags.php --tiles-set-id=1 --quiet --dry-run
+ */
+
+require_once( __DIR__ . '/../../Maintenance.php' );
+
+ini_set('display_errors', 'stderr');
+ini_set('error_reporting', E_ALL);
+
+class RemoveImapTags extends Maintenance {
+	static protected $dryRun = false;
+
+	private $maps;
+	private $mapsClientConfig;
+	private $slaveDBConfig;
+
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Removes from articles <imap /> tags for given tiles' set id.";
+		$this->addOption( 'dry-run', 'Dry run; make no changes', $required = false, $withArg = false );
+		$this->addOption(
+			'tiles-set-id',
+			'Specify a tiles\' set id which leads to a group of maps which <imap /> tags should get removed',
+			$required = true,
+			$withArg = true,
+			'tsi'
+		);
+	}
+
+	static public function isDryRun() {
+		return self::$dryRun;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)